vuepress/docs/.vuepress/v70.js

```diff
@@ -112,13 +112,25 @@ module.exports = {
                         title: 'Entando APIs',
                         path: path + 'consume/entando-apis.md'
                     },
+                    {
+                        title: 'Entando Identity Management System',
+                        path: path + 'consume/identity-management.md'
+                    },
                     {
                         title: 'Custom Resources',
                         path: path + 'consume/custom-resources.md'
                     },
                     {
-                        title: 'Entando Identity Management System',
-                        path: path + 'consume/identity-management.md'
+                        title: 'EntandoApp CRD',
```

Comment:
I like breaking them up but I'd suggest just linking to them from the parent Custom Resources page, rather than cluttering the left nav. I don't think someone is going to start by going directly to one of these - and then we don't need to find a user readable version of "EntandoApp CRD" for the left nav...